src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,197 @@
+import React, {useState, useEffect} from 'react'
+import {useAuth0} from '@auth0/auth0-react'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import CardContent from '@mui/material/CardContent'
+import CardHeader from '@mui/material/CardHeader'
+import InputBase from '@mui/material/InputBase'
+import Paper from '@mui/material/Paper'
+import useTheme from '@mui/styles/useTheme'
+import {TooltipIconButton} from '../Buttons'
+import useStore from '../../store/useStore'
+import {postIssue} from '../../utils/GitHub'
+import AddImage from '../../assets/icons/AddImage.svg'
+import LinkElement from '../../assets/icons/Link.svg'
+import Submit from '../../assets/icons/Submit.svg'
+import {useIsMobile} from '../Hooks'
+
+
+/**
+ * Note card
+ *
+ * @param {number} id note id
+ * @param {number} index note index
+ * @param {string} username username of the note author
+ * @param {string} title note title
+ * @param {string} avatarUrl user avatarUrl
+ * @param {string} body note body
+ * @param {string} date note date
+ * @param {number} numberOfComments number of replies to the note - referred to as comments in GH
+ * @param {boolean} expandedImage governs the size of the image, small proportions on mobile to start
+ * @param {boolean} isComment Comments/replies are formatted differently
+ * @return {object} React component
+ */
+export default function NoteCardCreate({
+  username = '',
+  avatarUrl = '',
+  expandedImage = true,
+  isComment = false,
+}) {
+  // eslint-disable-next-line no-unused-vars
+  const [expandImage, setExpandImage] = useState(expandedImage)
+  const repository = useStore((state) => state.repository)
+  const [title, setTitle] = useState('')
+  const [body, setBody] = useState('')
+  const isMobile = useIsMobile()
+  const {user, isAuthenticated} = useAuth0()
+
+  useEffect(() => {
+    if (isMobile) {
+      setExpandImage(false)
+    }
+  }, [isMobile])
+  const theme = useTheme()
+
+  /**
+   * create issue takes in the title and body of the note
+   *
+   */
+  function createIssue() {
```

Comment:
@oo-bldrs here I am creating an issue with the title + body pay lot as specified in the docs
https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#create-an-issue